src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -110,4 +111,10 @@ public void parseCommand_unrecognisedInput_throwsParseException() {
     public void parseCommand_unknownCommand_throwsParseException() {
         assertThrows(ParseException.class, MESSAGE_UNKNOWN_COMMAND, () -> parser.parseCommand("unknownCommand"));
     }
+
+    @Test
+    public void parseCommand_addTask() throws Exception {
+        assertTrue(parser.parseCommand(AddTaskCommand.COMMAND_WORD + " d/"
+                + "Buy fish D/2020-12-12 t/food") instanceof AddTaskCommand);
```

Comment:
Make use of the `TaskBuilder` class in `./testutil` to build a Task instead of hard-coding it.

Reply:
Only skeletal test case has been added will update test case again.